public/app/features/explore/state/reducers.ts

```diff
@@ -86,7 +86,11 @@ export const makeExploreItemState = (): ExploreItemState => ({
   initialized: false,
   queryTransactions: [],
   queryIntervals: { interval: '15s', intervalMs: DEFAULT_GRAPH_INTERVAL },
-  range: DEFAULT_RANGE,
+  range: {
```

Comment:
Related to earlier comments regarding `initializeExplore` action I think we could use a null range as default state instead and by that hopefully moving default range to one single place.